scripts/init/centos/gitea

```diff
@@ -27,7 +27,7 @@ NAME=gitea
 GITEA_HOME=/home/git/gitea
 GITEA_PATH=${GITEA_HOME}/$NAME
 GITEA_USER=git
-SERVICENAME="Gogs Go Git Service"
+SERVICENAME="Gitea Git with a cup of tea"
```

Comment:
Maybe add a dash between Gitea and Git, like so `Gitea - Git with a cup of tea`
Not sure what the guidelines for CentOS initrc is though.